### ISSUE TITLE
[Feature Request] console analogue to keybinds

### ISSUE THREAD

**Issue by tali713** [DFHack/stonesense#58]
Since keybinds are broken for mac, it would be nice to expose all action_$foo in stonesense/Keybinds.cpp as dfhack console commands, e.g. "stonesense rotate"->action_incrrotation.  This has the side effect of also making these commands available for dfhack key bindings.

I've been struggling to build dfhack locally or I'd test and submit a patch.

**Comment by lethosor** [comment]
What issues are you having building? Are you following the instructions at https://dfhack.readthedocs.io/en/latest/docs/Compile.html?

**Comment by tali713** [comment]
Yes indeed, and I get a "successful" compile, however when I launch dwarf fortress it declares (for each plugin) that the plugin couldn't be loaded.  I'm going to hop on irc in the next couple of days and see if I can figure out what silly mistake I am making.

**Comment by lethosor** [comment]
Maybe see what stderr.log says? It might be faster to troubleshoot on IRC, yeah.
